src/EnergyPlus/ConvectionCoefficients.hh

```diff
@@ -236,6 +236,7 @@ namespace ConvectionCoefficients {
         // Default Constructor
         RoofGeoCharacteristicsStruct() : BoundSurf(4)
         {
+            this->BoundSurf = Array1D<Vector>(4);
```

Comment:
This can be a `std::array<Vector, 4>` with no explicit construction here.

Reply:
Yeah I was just looking at why I even made that change...I'll fix it up.

Reply:
If we make this change to std::array, it requires changes the signature for CreateNewellAreaVector, which then hits a bunch of places in SurfaceGeometry, which will lead to more changes.  I can certainly go down that path, but it will much more likely start conflicting other branches.  For right now I'll hold off but if you feel it's a critical change then I'll be happy to do it here.

Reply:
Leave it for now.